validations/src/validations.rs

```diff
@@ -265,6 +265,13 @@ pub fn validate_mint_transaction(
         }
         .into())
     } else {
+        let first_pkh = mint_tx.outputs.first().unwrap().pkh;
```

Comment:
You will like https://docs.rs/itertools/0.9.0/itertools/trait.Itertools.html#method.all_equal